javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,7 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  for (let i = 0; i < words.length; i++) {
+    words[i] = words[i] + suffix;
+  }
+  return words;
+}
```

Comment:
We don't want to manipulate the `words` array directly. We just want to use this as a reference. I would've created a new array, and pushed the values into that new array. See my solution below.
```
function addSuffixToAll(words, suffix) {
  const result = [];
  for (let i = 0; i < words.length; i++) {
    result.push(words[i] + suffix);
  }
  return result;
}
```

Reply:
Updated. Thank you for letting me know.